Report/sections/1_analysis/Behavior.tex

```diff
@@ -0,0 +1,19 @@
+\chapter{Behavior}\label{chap:behavior}
+
+As concluded in the last chapter, learning how to program for the first time is hard.
+Therefore, a natural next question would be to ask how one could make it easier.
+That is the topic this chapter will examine. \\
+
+Earlier, in \chapref{chap:analysis}, we presented \textit{deliberate practice} as an optimal way to learn.
+However, merely knowing how to learn is not the same as learning. \\
+
+For any behavior to occur, three elements must be present. Namely, motivation, ability to perform a given behavior, and a prompt \cite{bjfoggBehaviorModelPersuasive2009}. If any of those are not present, the behavior will not occur. \\
+
+Often, increasing a user's ability is the best choice to optimizing behavior performance \cite{bjfoggBehaviorModelPersuasive2009}. Therefore, we should aim to make it simpler for users to program games.
```

Comment:
for optimizing

Reply:
also, program games specifically or program in general? Here you say games specifically, do we want to home in on games specifically this early on?